sdcm/nemesis/utils/common_ops.py

```
@@ -0,0 +1,85 @@
+
+        # How many SStables are going to be deleted
+        sstables_amount_to_destroy = int(len(all_files_to_destroy) * sstables_to_destroy_perc / 100)
+        runner.log.debug(
+            "SStables amount to destroy (%s percent of all SStables): %s",
+            sstables_to_destroy_perc,
+            sstables_amount_to_destroy,
+        )
+
+        destroyed_files = 0
+        while sstables_amount_to_destroy > 0:
+            file_for_destroy = random.choice(all_files_to_destroy)
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)